app/assets/stylesheets/registration3.scss

```diff
@@ -463,12 +463,60 @@ body {
   }
 }
 
+/* CTW additions */
+
+#main_inner .declaration-text {
+  background-color: #F5EEDD;
+  padding:10px;
+  margin:20px;
+  line-height:1.5;
+}
+
+
+.newui2020 #main_inner form ul.checkbox li h3 {
+  align-items:normal; /* checkbox aligned to top of text*/
+}
+
+#main_inner form ul.checkbox label {
+  width:calc(100% - 30px); /* make room for checkbox*/
+}
+
+div.error {
+    clear: both;
+    display: block;
+    width: 100%;
+    color: $red;
+    text-transform: none;
+    font-size: $base-font-size;
+    line-height: $base-line-height;
+    margin: 4px 0;
+    font-weight: normal;
+}
+
+#main {
+  form  {
+    .block-selector .block-selector { 
+      border: none; 
+      padding: 0
+    }
+  }
+}
+
 @import 'registration3-block-selector';
 
 @import 'registration3-mobile';
 @import 'registration3-new-mobile-ui';
 @import 'registration3-signature-capture';
+@import 'registration3-review';
 
 body.abr-page {
   @import 'abr-page';
-}
\ No newline at end of file
+}
+
+p.dummy-modified {
+    width:42px;
+}
+
+@import 'states/pa.scss';
+@import 'states/mi.scss';
+@import 'states/va.scss';
```

Comment:
Because of how the whitelabeling works, these should be imports of new CSS files (states/va3.scss) and the existing states/va.scss files should remain unchanged (no need to target .oldui) and the new CSS ruls will only get loaded via registration3.scss which only gets shown to users who should get the new ui.

Reply:
Have done